.travis.yml

```diff
@@ -83,6 +90,7 @@ script:
 after_failure:
   - echo "Logs from installation process come here / DEBUG LOGS"
   - cat /tmp/ci.log
+  - cat /tmp/tokumx.log
```

Comment:
Debug log not removed, or you want to keep it ? 
It's true that it would be nice to configure all integration programs to log in `/tmp/logs` and to have a `cat /tmp/logs/*` or something like this in `after_failure`.